_episodes/03-configuration.md

```diff
@@ -65,40 +65,26 @@ This file contains the information for:
 
 These settings are used to inform ESMValTool about your preference about
```

Comment:
This sentence, referring to "these settings" is a bit out of the blue now. Can we rephrase that?

Reply:
Sure we can, something like this?

Reply:
Great, thanks!